index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sum of 2 Numbers</title>
+</head>
+<body>
+    <div class="container">
+        <input type="number" id="n1" placeholder="First number..."><br>
+        <br>
+        <input type="number" id="n2" placeholder="Second Number..."><br>
+        <br>
+        <input type="button" value="Submit" onclick="submit()">
+        <br>
+        <div id="Add"></div>
```

Comment:
never start id with a capital letter. Just naming conventions
Instead of id="Add" it can be id="result"